coffee/drop.coffee

```diff
@@ -116,9 +116,20 @@ createContext = (options={}) ->
       addClass @content, "#{ drop.classPrefix }-content"
 
       if typeof @options.content is 'function'
-        @content.innerHTML = @options.content.call(@, @)
-        @on 'open', =>
-          @content.innerHTML = @options.content.call(@, @)
+        generateAndSetContent = () =>
+          # content function might return a string or an element
+          contentElementOrHTML = @options.content.call(@, @)
+
+          if typeof contentElementOrHTML is 'string'
+            @content.innerHTML = contentElementOrHTML
+          else if typeof contentElementOrHTML is 'object'
+            @content.appendChild(contentElementOrHTML)
```

Comment:
sorry, just found a bug -- this should probably first empty the `@content` element, _then_ `appendChild`. will push a PR for the fix.